pandas/tests/extension/test_string.py

```diff
@@ -44,32 +54,49 @@ def na_value():
 
 
 @pytest.fixture
-def data_for_grouping():
-    return StringArray._from_sequence(["B", "B", pd.NA, pd.NA, "A", "A", "B", "C"])
+def data_for_grouping(dtype):
+    return dtype.construct_array_type()._from_sequence(
+        ["B", "B", pd.NA, pd.NA, "A", "A", "B", "C"]
+    )
 
 
 class TestDtype(base.BaseDtypeTests):
     pass
 
 
 class TestInterface(base.BaseInterfaceTests):
-    pass
+    def test_view(self, data, dtype, request):
+        if isinstance(dtype, ArrowStringDtype):
+            reason = "Fails until implement, remove before merge"
+            mark = pytest.mark.xfail(reason=reason)
+            request.node.add_marker(mark)
```

Comment:
Do you know if you could make this a decorator? (or helper function) 
(to avoid repeating it below)

Reply:
we could, not sure I agree needed for 3 cases and also from the discussion in #31817, i'm not sure there is consensus on the best way to do it.